example_profile.ini

```diff
@@ -0,0 +1,19 @@
+; this is an example profile configuration for rolling pin
+; technically any value in the example.ini file can be used here
+; but it is mostly for overriding and defaulting more execution values
+
+[deploy]
+; this sets a default set of hosts to apply to (default for the '-h' argument)
+hosts = apps
+
+; this sets a default set of components to deploy (default for the '-c' argument)
+components = rollingpin
+
+; this sets the default set of hosts to restart (default for the '-r' arguments)
+restarts = all
+
+[harold]
+; per profile override of the '[harold]' section in the primary config (see the
+; configuration there)
+base-url =
+secret =
```

Comment:
This is meant to demonstrate overriding from the base config (which has empty definitions too).  If it is more confusing, I can remove it...